test-case/check-runtime-pm-status.sh

```diff
@@ -82,8 +84,6 @@ do
 
     for i in $(seq 1 $loop_count)
     do
-        # set up checkpoint for each iteration
-        setup_kernel_check_point
```

Comment:
please don't remove the setup_kernel_check_point in the loop, because we only want to collect logs from the latest (success or failure) iteration, to reduce log size.

We are OK with only the changes in line 53